flowmachine/tests/test_query_object_construction.py

```diff
@@ -355,3 +403,30 @@ def test_invalid_sampling_params_raises_error(sampling, message):
     with pytest.raises(ValidationError, match=message) as exc:
         _ = FlowmachineQuerySchema().load(query_spec)
     print(exc)
+
+
+@pytest.mark.parametrize("query_type", ["daily_location", "modal_location"])
+def test_unmatching_spatial_unit_raiese_error(query_type):
+    query_spec = {
+        "query_kind": query_type,
```

Comment:
'query_kind' here should always be "location_visits" - I'm guessing you intended to parametrize the 'locations'?